### DISCUSSION TITLE
duda código

### DISCUSSION THREAD

**Discussion by rominaarancibia** [IIC2133-PUC/2024-1#113] [Tarea 1] [answered]
Hola! Tengo una pequeña duda de un casting que realizo por lo que quería preguntar a que ayudante podría mandarle un mail ya que debo mostrar un fragmento de mi código. En verdad son dos líneas solamente me quiero asegurar que no es problemático.
Gracias

**Comment by benjavicente** [comment]
¿Es técnica la pregunta? Puedes de tratar de dar un ejemplo similar, pero no cercano a lo que estás haciendo, para entender tu duda y a donde vas 🤔 

Si es muy complicada la duda, mándamela por correo y la comparto con el resto de los ayudantes, para tener donde es lo que quieres reviar.

**Reply by rominaarancibia** [reply, marked as answer]
Te mandé un mail. muchas graciass
